x-pack/plugins/enterprise_search/public/applications/app_search/components/engines/components/empty_meta_engines_state.test.tsx

```diff
@@ -21,7 +23,7 @@ describe('EmptyMetaEnginesState', () => {
 
     expect(wrapper.find('h3').text()).toEqual('Create your first meta engine');
     expect(wrapper.find(EuiButton).prop('href')).toEqual(
-      expect.stringContaining('/meta-engines-guide.html')
+      expect.stringContaining(`${docLinks.appSearchMetaEngines}`)
```

Comment:
```suggestion
      expect.stringContaining(docLinks.appSearchMetaEngines)
```
No need for a template literal here